doc/user_guide/transformations.rst

```diff
@@ -244,6 +244,20 @@ can be found in the API-specific sections).
     :members: apply
     :noindex:
 
+
+####
+
+.. autoclass:: psyclone.transformations.KernelGlobalsToArguments
```

Comment:
Please could you move this section so that it is alphabetical order with respect to the other transformations.

Reply:
Done